Index.html

```diff
@@ -21,54 +21,8 @@
     <link href="styles/Main.css" rel="stylesheet">
     <link rel="manifest" href="site.webmanifest" />
     <meta name="msapplication-TileColor" content="#da532c" />
-    <meta name="theme-color" content="#ffffff" />\
-    <style>
-      h1{ 
-        background: rgb(2,0,36);
-        background: linear-gradient(180deg, rgba(2,0,36,1) 0%, rgba(9,121,110,0.36458333333333337) 24%, rgba(23,111,163,1) 100%);
-      }
-      h1,h2,h3,h4,h5,h6 {
-        font-family: 'Space Mono', monospace;
-      }
-      p {
-        font-family: 'Asap', sans-serif;
-      }
-      body a{
-        text-decoration: none;
-      }
-      body a:hover{
-        text-decoration: underline;
-      }
-      ol{
-       text-transform: uppercase;
-       list-style-type: upper-roman;
-      }
-      iframe{
-        border-top-right-radius: .5cm;
-        border-bottom-left-radius: .5cm;
-      }
-      ul{
-        text-decoration: none;
-        text-emphasis: Bold;
-        text-transform: uppercase;
```

Comment:
Again, the rule you have here selects all unordered lists, but uppercase was only to apply to navigation links